docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,499 @@
+
+    2.   Provide the required information such as target, local paths to boot and platform image, and so on.
+
+         1.   **Profile name**: Enter a profile name, for example, Rpi4_Headed_Profile.
+         2.   **Target**: Select the board and image type from the drop-down list. For example, RPI4_HEADED.
+         3.   **Boot image**: Browse for the Tizen Boot image available on your hard drive. You must select proper images depending on the board. For more information, see [Downlaod Guide](#download-binaries).
+         4.   **Platform image**: Browse for the Tizen Platform image available on your hard drive. You must select proper images depending on the board. For more information, see [Downlaod Guide](#download-binaries).
+         5.   **RPM bundles**: 
+              -   Add the bundle directory to the Bundles panel.
+              -   Use this to install Wi-Fi plugin. Download using [Install Drivers](#install-drivers). Unzip and add path to 32-bit rpm or 64-bit rpm according to image to install.
+              -   Leave blank if no rpm installation required.
+         6.   **WiFi**: Enter the SSID and password to connect automatically to Wi-Fi. (Wi-Fi plugin installation is required). If Wi-Fi or ethernet is not required, you can clear the respective check box.
+         7.   **Ethernet**:
+              -   **IP**: 192.168.1.11
+              -   **Gateway**: 192.168.1.2
+              -   **Subnet**: 255.255.255.0
+              -   **Primary DNS Server**: 127.0.0.1
+
+     3. Click **OK** to save.
+
+2. Select the SD card on which you want to burn Tizen:
+      1. Insert your SD Card into your computer.
+      2. Select the appropriate SD Card from the drop-down list. For example, `/dev/sdx` on Ubuntu and `F:` on Windows.
+
+      > [!NOTE]
+      > Currently, the USB SD card reader is only supported. The internal PC SD slot is not supported.
+      > Also, if the **Select Drive** does not show anything, detach and insert the SD card again into the computer, or close and restart IoT Setup Manager.
+
+3. Burn Tizen on your SD Card:
+      1. Click **Burn** to burn Tizen to your SD Card. This takes about two minutes to complete and you may have to enter your supervisor password, which is required to access the SD card.
+	
+#### Precaution while flashing image
+
+-   While creating or editing a profile, ensure that the boot images, platform images, and the RPMs (if selected) match the target device.
+-   Ensure that there are no white spaces in the boot image, platform image, and the RPM folder paths.
+-   Place RPMs in a separate directory, which does not contain any other files or directories and select the directory path for creating the profile.
+
+#### IoT Setup Manager features
+
+-   **Saved Profiles:** You can save and retrieve the configuration profiles for easy flashing.
+-   **Three Steps Flashing:**
+    1.  Select the profile.
+    2.  Select your SD Card.
+    3.  Click **Burn**.
```

Comment:
I think we can remove this "IoT setup manager features" section as the context is duplicated. Would you please let me know your opinion?